.eslintrc.json

```diff
@@ -19,6 +19,7 @@
   },
   "plugins": ["simple-import-sort", "no-only-tests", "unused-imports"],
   "rules": {
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
It's so easy to abuse this. I prefer the ugly comments required in each instance